rollup.config.js

```diff
@@ -38,13 +39,11 @@ export default {
 	plugins: [
 		svelte({
 			// enable run-time checks when not in production
-			dev: !production,
-			// we'll extract any component CSS out into
-			// a separate file - better for performance
-			css: css => {
-				css.write('bundle.css');
-			}
+			dev: !production
```

Comment:
If we're updating this for 7.0, this won't work anymore. Should be
```suggestion
			compilerOptions: {
			  dev: !production
			}
```